### ISSUE TITLE
Platform Dependent may not work for bundle

### ISSUE THREAD

**Issue by coolacid** [steakknife/oui#17]
When trying to do a bundle install for my logstash plugin, it wanted to kill the jdbc-sqlite3 requirement.

I ended up solving this by adding a development_dependency for jdbc-sqlite3 in my project (we only use JRUBY)

I'm wondering if adding a dev_dep to this Gem with platform dependent checks would work as an upstream item.

**Comment by steakknife** [comment]
Fixed it.  Separate gems per platform with hard deps on their respective drivers.

Hope that’s cool.  If not, let me know.

Barry

**Comment by steakknife** [comment]
Btw the ext hack would be neat for soft dependencies *on the same platform*.  Useful +1.

**Comment by steakknife** [comment]
Released 1.2.6.  One gem for you, one for me. ;)

**Comment by steakknife** [comment]
**JRuby**
```shell
$ gem install oui-offline
Fetching: sequel-4.18.0.gem (100%)
Successfully installed sequel-4.18.0
Fetching: jdbc-sqlite3-3.8.7.gem (100%)
Successfully installed jdbc-sqlite3-3.8.7
Fetching: oui-offline-1.2.6-java.gem (100%)
Oui!
Successfully installed oui-offline-1.2.6-java
3 gems installed
```

**Ruby**
```shell
$ gem install oui-offline
Fetching: sqlite3-1.3.10.gem (100%)
Building native extensions.  This could take a while...
Successfully installed sqlite3-1.3.10
Fetching: sequel-4.18.0.gem (100%)
Successfully installed sequel-4.18.0
Fetching: oui-offline-1.2.6.gem (100%)
Oui!
Successfully installed oui-offline-1.2.6
3 gems installed
```
